frontend/src/metabase-lib/lib/metadata/Database.js

```diff
@@ -93,15 +93,6 @@ export default class Database extends Base {
 
   /** Returns a database containing only the saved questions from the same database, if any */
   savedQuestionsDatabase(): ?Database {
-    const database = this.metadata
-      .databasesList()
-      .find(db => db.is_saved_questions);
-    if (database) {
```

Comment:
Why is this no longer relevant? This filtered out questions that weren't part of the same underlying database.

Reply:
Whoops I totally missed that. Fixing now...